app/main.py

```diff
@@ -1,6 +1,41 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set before running the script. If it's not set, the script will not be able to authenticate with the Weather API.